<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddCourse</class>
 <widget class="QDialog" name="AddCourse">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>457</width>
    <height>348</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="courseName">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>40</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="courseType">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>80</y>
     <width>75</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>必修课</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>选修课</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>40</y>
     <width>54</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>课程名字</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>80</y>
     <width>54</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>课程性质</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>120</y>
     <width>54</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>学生人数</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>160</y>
     <width>54</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>学分</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>200</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>上课时间</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>200</y>
     <width>54</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>节数</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="studentNumber">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>120</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="credit">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>160</y>
     <width>75</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>1.0</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2.0</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>3.0</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4.0</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>5.0</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>6.0</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>7.0</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="classTime">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>200</y>
     <width>75</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>5</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>270</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>270</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QComboBox" name="classWeek">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>200</y>
     <width>75</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>星期一</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>星期二</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>星期三</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>星期四</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>星期五</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>星期六</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>星期天</string>
    </property>
   </item>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
